Files/LUISApp.json

```diff
@@ -142,17 +142,17 @@
       "entities": []
     },
     {
-      "text": "list new idea",
+      "text": "list new ideas",
       "intent": "Idea.List",
       "entities": []
     },
     {
-      "text": "show me idea in progress",
```

Comment:
This is not a typo. It was specially created for better compatibility in case that users may misspell it.
You can find "show me ideas in progress" on [line 135](https://github.com/OfficeDev/Office-365-Huddle-Templates/blob/master/Files/LUISApp.json#L135)

Reply:
Rolled back